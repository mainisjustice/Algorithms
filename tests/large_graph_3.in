20
0 18 15 17 0 19 13 16 18 20 14 2 19 18 4 23 3 21 5 22
18 0 21 0 2 10 1 10 4 22 11 3 3 17 5 6 6 13 0 10
15 21 0 11 23 4 24 3 7 8 14 24 14 0 24 9 2 23 8 16
17 0 11 0 5 8 19 19 7 15 10 19 14 24 3 4 12 0 4 11
0 2 23 5 0 21 21 21 7 5 0 0 15 5 24 4 8 11 13 4
19 10 4 8 21 0 4 4 17 3 3 12 15 16 0 8 1 22 0 12
13 1 24 19 21 4 0 16 5 10 18 11 11 6 12 2 23 13 19 18
16 10 3 19 21 4 16 0 11 23 13 17 14 17 20 0 24 18 24 2
18 4 7 7 7 17 5 11 0 0 12 3 11 11 16 7 22 8 20 12
20 22 8 15 5 3 10 23 0 0 20 14 22 17 5 3 14 16 4 24
14 11 14 10 0 3 18 13 12 20 0 10 20 12 16 10 20 11 8 14
2 3 24 19 0 12 11 17 3 14 10 0 4 4 21 0 19 5 0 13
19 3 14 14 15 15 11 14 11 22 20 4 0 5 16 13 4 9 7 11
18 17 0 24 5 16 6 17 11 17 12 4 5 0 22 20 8 6 4 4
4 5 24 3 24 0 12 20 16 5 16 21 16 22 0 14 21 16 5 2
23 6 9 4 4 8 2 0 7 3 10 0 13 20 14 0 18 18 22 11
3 6 2 12 8 1 23 24 22 14 20 19 4 8 21 18 0 2 18 19
21 13 23 0 11 22 13 18 8 16 11 5 9 6 16 18 2 0 10 4
5 0 8 4 13 0 19 24 20 4 8 0 7 4 5 22 18 10 0 10
22 10 16 11 4 12 18 2 12 24 14 13 11 4 2 11 19 4 10 0