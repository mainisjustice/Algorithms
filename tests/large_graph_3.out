0 5 5 7 7 4 6 8 5 7 7 2 6 6 4 8 3 5 5 6 
5 0 8 7 2 5 1 8 4 6 8 3 3 7 5 3 6 8 10 6 
5 8 0 10 9 3 7 3 7 6 6 7 6 9 7 9 2 4 8 5 
7 7 10 0 5 8 6 7 7 7 10 9 10 8 3 4 9 9 4 5 
7 2 9 5 0 7 3 6 6 5 10 5 5 5 6 4 8 8 9 4 
4 5 3 8 7 0 4 4 9 3 3 6 5 9 8 6 1 3 7 6 
6 1 7 6 3 4 0 8 5 5 7 4 4 6 6 2 5 7 9 7 
8 8 3 7 6 4 8 0 10 7 7 10 9 6 4 10 5 6 9 2 
5 4 7 7 6 9 5 10 0 10 12 3 7 7 9 7 8 8 10 10 
7 6 6 7 5 3 5 7 10 0 6 9 8 8 5 3 4 6 4 7 
7 8 6 10 10 3 7 7 12 6 0 9 8 12 11 9 4 6 8 9 
2 3 7 9 5 6 4 10 3 9 9 0 4 4 6 6 5 5 7 8 
6 3 6 10 5 5 4 9 7 8 8 4 0 5 8 6 4 6 7 9 
6 7 9 8 5 9 6 6 7 8 12 4 5 0 6 8 8 6 4 4 
4 5 7 3 6 8 6 4 9 5 11 6 8 6 0 7 7 6 5 2 
8 3 9 4 4 6 2 10 7 3 9 6 6 8 7 0 7 9 7 8 
3 6 2 9 8 1 5 5 8 4 4 5 4 8 7 7 0 2 8 6 
5 8 4 9 8 3 7 6 8 6 6 5 6 6 6 9 2 0 10 4 
5 10 8 4 9 7 9 9 10 4 8 7 7 4 5 7 8 10 0 7 
6 6 5 5 4 6 7 2 10 7 9 8 9 4 2 8 6 4 7 0 